### DISCUSSION TITLE
MySQL (Specifying root password)

### DISCUSSION THREAD

**Discussion by hassanaleem** [slimtoolkit/slim#460] [Q&A]
I am trying to generate a slim image and seccomp profile for MySQL image from DockerHub (https://hub.docker.com/_/mysql). I am getting the following error:

![1](https://user-images.githubusercontent.com/59873682/214135684-72198f8a-05b6-44d5-b6d0-4452ff8fa4a4.png)
![2](https://user-images.githubusercontent.com/59873682/214135716-94f4b2cc-605a-4436-846f-632e6e0333be.png)

Can someone please help

**Comment by kcq** [comment]
You can use the `--env` command flag when you `slim build`. It's similar to the `--env` / `-e` flag in `docker run`, so your call might look something like this (note the main app is `slim` instead of `docker-slim`): `slim build --env MYSQL_ROOT_PASSWORD=_YOUR_PASSWORD_HERE_ --target mysql:debian --copy-meta-artifacts ./`
